### DISCUSSION TITLE
how to stop netbox service ?

### DISCUSSION THREAD

**Discussion by Jason-MK** [netbox-community/netbox#7237] [Q&A]
Hello,

I'm trying to migrate my database to a new machine so I need to stop Netbox and each time I type 

sudo systemctl stop netbox netbox-rq

I get this message 

Failed to stop netbox.service: Unit netbox.service not loaded.
Failed to stop netbox-rq.service: Unit netbox-rq.service not loaded

Can anyone help me please in this I'm using debian 10 

Best Regards

**Comment by candlerb** [comment]
What version of Netbox are you running?  If it's an older installation, the documentation used to say to use supervisord instead of systemd.

**Reply by Jason-MK** [reply]
I'm using this version v3.0.2

**Comment by candlerb** [comment]
If the version you're trying to stop was a fresh install of v3.0.2, and you followed the [installation instructions](https://netbox.readthedocs.io/en/stable/installation/4-gunicorn/#systemd-setup) fully, then you would have had systemd unit files.

If you didn't, then all I can say is "whatever method you used to start netbox, you need to undo"

For example, if you're running the development server with `runserver` as described [here](https://netbox.readthedocs.io/en/stable/installation/3-netbox/#test-the-application), then you need to find that window and hit Ctrl-C, or you need to find the process ID and kill it.
